src/A1_csv_to_json.py

```diff
@@ -302,6 +302,10 @@ def create_json_from_csv(input_directory, filename, parameters):
         index_col=0,
     )
 
+    # check wether parameter maximumCap is availavle                             #todo in next version: add maximumCap to hardcoded parameter list above
+    if "maximumCap" in df.index:
+        parameters.append("maximumCap")
```

Comment:
```suggestion
   new_parameter = "maximumCap"
    if new_parameter  in df.index:
        parameters.append(new_parameter )
   else:
       logging.warning("You are not using the parameter %s for asset group %s, which allows setting a maximum capacity for an asset that is being capacity optimized (Values: None/Float). In the upcoming version of the MVS, this parameter will be required.", new_parameter, filename) 
```
Probably this would require that filename is in a specific list of csv files, ie. the "energyXY".csv.